src/routes/[nav]/+page.svelte

```diff
@@ -1,9 +1,12 @@
 <script lang="ts">
 	import { extractH1AndContent } from '$lib';
 	import { Avatar } from '@skeletonlabs/skeleton';
+	import { DateTime } from 'luxon';
 	import type { PageData } from './$types';
 	export let data: PageData;
-
+	$: committeePageLastEditedAtFormatted = DateTime.fromISO(
+		data.committeePage!.lastEditedAt
+	).toFormat('lastEditedAtFormat');
```

Comment:
This string results in nonsensical formatting
```suggestion
	).toFormat(lastEditedAtFormat);
```

Reply:
An error